buildscripts/docker-entrypoint.sh

```diff
@@ -78,4 +78,10 @@ docker_secrets_env
 ## Wait for all the hosts to come online.
 docker_wait_hosts "$@"
 
+# allow the container to be started with `--user`
+if [ "$1" = 'minio' -a "$(id -u)" = '0' ]; then
+  chown -R minio .
```

Comment:
This should be done for the "/export" - also looks like the tabbing is off.. Also don't need to provide this option.  Let's by default move to user style approach.  `su-exec` is not needed. 

All directories provided and local to this instance should chown itself. It would require properly parsing minio command line though.